subprojects/packagefiles/sdl2/meson.build

```diff
@@ -184,6 +184,7 @@ check_functions = [
 # check token HAVE_SEM_TIMEDWAIT
   ['HAVE_INOTIFY_INIT', 'inotify_init', '#include<sys/inotify.h>'],
   ['HAVE_INOTIFY_INIT1', 'inotify_init1', '#include<sys/inotify.h>'],
+  ['HAVE_DLOPEN', 'dlopen', '#include<dlcfn.h>'],
```

Comment:
All the dlopen logic came from https://github.com/libsdl-org/SDL/blob/7967c97618fba7326028005e72ead740fa6e6589/cmake/sdlchecks.cmake#L32